packages/grid/x-data-grid-premium/src/DataGridPremium/DataGridPremium.tsx

```diff
@@ -60,6 +60,16 @@ interface DataGridPremiumComponent {
   propTypes?: any;
 }
 
+/**
+ *
+ * Demos:
+ *
+ * - [DataGridPremium](https://mui.com/x/react-data-grid/demo/)
```

Comment:
little question here: the API pages link to the overview page and not the demo page directly. Should we follow this approach with these links as well, or would it be better to directly link to the Demo page?

Reply:
Yes, the hash of the link in API page is broken, it should redirect to https://mui.com/x/react-data-grid/#commercial-versions I assume the choice of the section "Commercial versions" is to ensure people discovering it knows about the different plans.

For me linking to demo looks good :+1: